app/src/main/java/com/github/se/gomeet/ui/mainscreens/profile/SettingsScreen.kt

```diff
@@ -0,0 +1,195 @@
+package com.github.se.gomeet.ui.mainscreens.profile
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.navigation.compose.rememberNavController
+import com.github.se.gomeet.R
+import com.github.se.gomeet.ui.navigation.BottomNavigationMenu
+import com.github.se.gomeet.ui.navigation.NavigationActions
+import com.github.se.gomeet.ui.navigation.Route
+import com.github.se.gomeet.ui.navigation.TOP_LEVEL_DESTINATIONS
+import com.github.se.gomeet.ui.theme.DarkCyan
+
+@Composable
+fun SettingsScreen(
+    nav: NavigationActions, /*userViewModel: UserViewModel*/
+) {
+  Scaffold(
+      modifier = Modifier.testTag("SettingsScreen"),
+      topBar = {
+        Column {
+          Text(
+              text = "Settings",
+              modifier = Modifier.padding(top = 15.dp, start = 15.dp, end = 18.dp, bottom = 0.dp),
```

Comment:
We should avoid Hard-Coded padding values, as it may lead to undefined behaviors on different screen sizes. It should be good for now as these are small values, but if they are accumulated to other paddings this may not display what is expected.
Do you think we should modify the paddings now, or do it for the whole project on a separate PR?

Reply:
I think for now it is fine as other developers have tested the UI on other emulators including a tablet emulator so I think we are good for now.